src/pages/Distinction/DistinctionTable/FormEditDistinctionTypes.tsx

```diff
@@ -135,7 +135,7 @@ const FormEditDistinctionTypes: React.FC<FormEditDistinctionTypesProps> = () =>
               name="inputName"
               value={title}
               onChange={(event) => {
-                if(event.target.value.length < 250)
+                if(event.target.value.length < 200)
```

Comment:
it would be better to set the magic number 200 to a const variable

Reply:
changed